### PR TITLE
Updated colors to support Mattermost themes

### PR DESCRIPTION
**Pull request by harshilsharma63**
#### Summary
Updated colors to support Mattermost themes

#### Ticket Link
Fixes https://mattermost.atlassian.net/browse/MM-57748

### CODE REVIEW

File: webapp/src/components/surveyPost/linearScaleQuestion/style.scss [hunk 1]
```diff
@@ -3,29 +3,24 @@
...
+            opacity: 0.75;
         }
 
         .indents {
             display: flex;
             width: min-content;
+            max-width: 100%;
             flex-direction: row;
             border-radius: 16px;
             background: rgba(var(--button-bg-rgb), 0.08);
```
Comment (abhijit-singh) [inline webapp/src/components/surveyPost/linearScaleQuestion/style.scss:26]: background: rgba(var(--button-bg-rgb), 0.12);

**Review by abhijit-singh — CHANGES_REQUESTED** [review]
Thanks @harshilsharma63! I added a comment separately and rquesting one more change:
In `webapp/src/components/surveyPost/linearScaleQuestion/style.scss`, line 35, under  `&.disabled`, the color variable set might be incorrect. I think we can update it to `background: var(--center-channel-color-08);`

File: webapp/src/components/surveyPost/linearScaleQuestion/style.scss (hunk 1 above)
Comment (harshilsharma63) [reply webapp/src/components/surveyPost/linearScaleQuestion/style.scss]: Done!

**Comment by harshilsharma63** [comment]
> Thanks @harshilsharma63! I added a comment separately and rquesting one more change: In `webapp/src/components/surveyPost/linearScaleQuestion/style.scss`, line 35, under `&.disabled`, the color variable set might be incorrect. I think we can update it to `background: var(--center-channel-color-08);`

Done!

**Comment by harshilsharma63** [comment]
@abhijit-singh can please please try this out?

**Review by abhijit-singh — APPROVED** [review]
Looks good, thanks!

**Review by Rajat-Dabade — APPROVED** [review]

**Pull request merged**